packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/datebox.mask.tests.js

```diff
@@ -1266,6 +1266,45 @@ module('Date AM/PM Handling', setupModule, () => {
     });
 });
 
+module('TimeZone Handling', setupModule, () => {
+    module('regex tests', () => {
```

Comment:
This module seems to be excess for me

Reply:
[3c3e99e](https://github.com/DevExpress/DevExtreme/pull/27770/commits/3c3e99eba324815ee381b3c900d53123cfa91ac8)